### ISSUE TITLE
SyncBatchNormalization is not supported by ONNX but used in CenterNet HourGlass104 512x512 

### ISSUE THREAD

**Issue by Mypathissional** [tensorflow/models#10629]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [x] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [x] I checked to make sure that this issue has not been filed already.

## 2. Describe the bug
Hi, the saved model used for CenterNet HourGlass104 512x512 have SyncBatchNormalization that are not supported by ONNX. Is it possible to have new saved models which have normal BatchNormalization layers?
```
strategy = tf.distribute.MirroredStrategy()
with strategy.scope():
  net = tf.keras.Sequential()
  net.add(tf.keras.layers.Conv2D(2,3))
  net.add(tf.keras.layers.experimental.SyncBatchNormalization())
net.build((1,30,30,2))
net.save("~/Desktop/conv_block")
```
n also obtain the TensorFlow version with:
I have aksed about this also in https://github.com/tensorflow/models/issues/10629
2. TensorFlow 2.0
v2.8.0-rc1-32-g3f878cff5b6 2.8.0

**Comment by saberkun** [comment]
File a bug to Keras or TF github repo? This models repo cannot help fix the infrastructure problems.
